cirq/circuits/optimization_pass.py

```diff
@@ -41,8 +41,11 @@ def __init__(self, clear_span: int, clear_qubits: Iterable['cirq.Qid'],
                 with each affected moment.
             new_operations: The operations to replace the cleared out
                 operations with.
+            preserve_moments: Whether to keep Moments intact instead of
+                flattening them
```

Comment:
Since, as you said, moments will still be flattened anyway, is this description a bit misleading?

Reply:
updated description